Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -138,7 +144,11 @@ void TextureCacheBase::Cleanup(int _frameCount)
   TexAddrCache::iterator tcend = textures_by_address.end();
   while (iter != tcend)
   {
-    if (iter->second->frameCount == FRAMECOUNT_INVALID)
+    if (iter->second->tmem_only)
+    {
+      iter = InvalidateTexture(iter);
```

Comment:
The pr does this in InvalidateTexture(), because i couldn't write nice and clean code for this. I think it's possible for a texture to be bound to different tmem slots, at least i don't want to trust that there aren't. So i'd need to go through all of them, and check if any of them is bound. This is also the reason why i don't want to store the binding in the texture cache entry. The only way i could come up with was a bool and a loop going through the array, since again i'd need the index and the element to check this.